### ISSUE TITLE
Chat actions appear disabled by default, poor contrast on buttons actually disabled

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#295885]
Using the default dark modern theme:

1. These look like they're all disabled (but they're not)
2. The action that's actually disabled (on the far right) does not have sufficient contrast

<img width="641" height="198" alt="Image" src="https://github.com/user-attachments/assets/2f412e2a-3129-411e-b2c9-d6dec68f8cf1" />

from https://github.com/microsoft/vscode/pull/294351

**Comment by daviddossett** [comment]
hm yeah this is worse in dark modern than the new dark theme. Still need some color tweaking:

<img width="1256" height="294" alt="Image" src="https://github.com/user-attachments/assets/e83974b5-b1ed-4fcb-8421-542aafc79d4c" />

 Mostly because we use this heavy background color for inputs on Dark Modern. I'll see if it's better to tweak that or to revert.

FWIW, the goal is to make the input feel like not everything is shouting at you. Take the Codex extension as inspiration:

<img width="990" height="306" alt="Image" src="https://github.com/user-attachments/assets/19a1745f-8bd2-46af-b588-dd63ecd4bed6" />
